### ISSUE TITLE
Move project button does not work for accessibility users

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1170]
Projects cannot be moved via keyboard only in the Project Administrator view as the move button is not tab selectable, the move-projects icon also needs alt-text/aria-label or should be hidden from aria users due to  inability to function. Alternative capability may need to be identified to support accessibility.

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/1342

**Comment by sudo-may** [comment]
As a part of this ticket also added support for key-based sorting of the subjects, badges and projects on the Progress&Ranking pages

**Comment by mosefer** [comment]
moving projects down in the order seems to inconsistently function

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/1371

**Comment by mosefer** [comment]
:+1:
